docs/content/en/docs/Architecture/meta.md

```diff
@@ -22,9 +22,21 @@ Every component is extensible and modular such as it can be customized and repla
 
 ### Internal components
 
-Kairos encompasses several components, some externally, most notably:
-
+Kairos encompasses several components, external and internal.
+
+Internal:
+- [kairos](https://github.com/kairos-io/kairos) is the main repository, building the `kairos-agent` and containing the image definitions which runs on our CI pipelines.
+- [immucore](https://github.com/kairos-io/immucore) is the immutability management interface.
+- [AuroraBoot](https://github.com/kairos-io/AuroraBoot) is the Kairos Node bootstrapper
+- [elemental-cli](https://github.com/kairos-io/elemental-cli) manages the installation, reset, and upgrade of the Kairos node.
```

Comment:
We ened to rename this asap :D

Reply:
yep indeed... we have https://github.com/kairos-io/kairos/issues/711 in the sprint :) , I would have waited for https://github.com/kairos-io/kairos/issues/711, but this list in the docs was not matching the truth since a long time already, so thought to updated it